src/hooks/use-aggrid-sort.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useCallback, useState } from 'react';
+
+export const SORT_WAYS = {
+    asc: 1,
+    desc: -1,
+};
+
+export const useAgGridSort = (dataKeyToSortKey, initSortConfig) => {
```

Comment:
this hooks works when sorting on one row only
either fix it when sorting on several row 
or disable multisort and add a comment to warn developers using this hook

Reply:
I talked with PO and we desactivated multisort . thank you